exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py

```diff
@@ -70,6 +70,7 @@ class OTLPSpanExporter(
         headers: Headers to send when exporting
         timeout: Backend request timeout in seconds
         compression: gRPC compression method to use
+        max_attr_value_length: Max length string attribute values can have. Set to None to disable.
```

Comment:
I think it applies to `sequence` attributes too. Although the behavior for `zipkin` and `jaeger` truncates the sequence in a stringified form, so not sure when this would happen for `otlp`.

Reply:
But if I remember it incorrectly, happy to do it here.

Reply:
The proposed addition to spec recommends to truncate individual values in sequence types. The proposal has not been merged yet but judging from the discussion I don't see it changing drastically.  (https://github.com/open-telemetry/opentelemetry-specification/pull/1130)

Either way we can use it as a general guidance for this feature when implementing in exporters. Once it is merged, we'd likely want to implement these limits directly in the span or attribute implementations.

Reply:
Yeah truncating individual values makes much more sense.